v2/protocol/http/utility.go

```diff
@@ -0,0 +1,31 @@
+/*
+ Copyright 2022 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package http
+
+import (
+	"context"
+	nethttp "net/http"
+
+	"github.com/cloudevents/sdk-go/v2/binding"
+	"github.com/cloudevents/sdk-go/v2/event"
+)
+
+// NewEventFromHttpRequest returns an Event.
+func NewEventFromHttpRequest(req *nethttp.Request) (*event.Event, error) {
+	msg := NewMessageFromHttpRequest(req)
+	return binding.ToEvent(context.Background(), msg, nil)
+}
+
+// NewEventFromHttpResponse returns an Event.
+func NewEventFromHttpResponse(resp *nethttp.Response) (*event.Event, error) {
```

Comment:
```suggestion
func NewEventFromHTTPResponse(resp *nethttp.Response) (*event.Event, error) {
```

Reply:
I found `NewMessageFromHttpRequest` using camel case so I tried to track against that, happy to follow convention.